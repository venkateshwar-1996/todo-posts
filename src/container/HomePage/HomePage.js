import React, { useContext } from "react";
import Container from "react-bootstrap/Container";
import DisplayPosts from "../../component/DisplayPosts";
import { DataContext } from "../../context/dataContext";

const HomePage = () => {
const context = useContext(DataContext);
  return (
    <>
      <Container data-testid='homepage'>
        <DisplayPosts isLoading={context.loading} />
      </Container>
    </>
  );
};

export default HomePage;
