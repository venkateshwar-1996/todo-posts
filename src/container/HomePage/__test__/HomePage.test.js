import React from 'react';
import { render } from '@testing-library/react';
import HomePage from '../HomePage';

test('renders Homepage without crash', () => {
  const { getByTestId } = render(<HomePage />);
  expect(getByTestId('homepage')).toMatchSnapshot();
});
