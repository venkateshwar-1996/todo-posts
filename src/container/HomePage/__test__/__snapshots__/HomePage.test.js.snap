// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Homepage without crash 1`] = `
<div
  class="mt-4 container"
  data-testid="homepage"
>
  <div
    class="list-group list-group-flush"
  />
</div>
`;
